doc/setup.rst

```diff
@@ -198,6 +198,55 @@ To help expose these tracepoints (and any additional one we might require in
 the future) as trace events, an external module is required and is provided
 under the name of "lisa" in $LISA_HOME/tools/kmodules/lisa
 
+Pre-requisites
+.................
+
+CFI
+~~~~~
+
+First of all, building the module outside the kernel will not work if the kernel
+has CFI (Control-Flow Integrity) enabled as it will not allow loading of outside
+modules. Android kernels come with CFI by default so it will have to be disabled
+in the config (``CONFIG_CFI_CLANG``).
+If you want to use the module with a CFI-enabled kernel, follow the steps under
+"Integrating the module in your kernel tree" below.
```

Comment:
use target and reference:
https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets